src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -3743,6 +3743,31 @@ public async Task CanExecuteRequestDelegateWithResultsExtension()
         Assert.Equal(@"""Hello Tester. This is from an extension method.""", decodedResponseBody);
     }
 
+    [Fact]
+    public async Task CanExecuteRequestDelegateWithTypedResultsExtension()
```

Comment:
Can you move this test over to this file: https://github.com/dotnet/aspnetcore/blob/f876ff14f8d4d28b410f294f2ec92b3aaf619060/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.Responses.cs?

Context: we now have a compile-time source generator for minimal APIs and placing your test in the file above will validate we have the same behavior between the compile-time and runtime implementations.

Reply:
Could you give me some guidance on how to provide an extra class to the 'source' of my test? 
Basically I need to use the following source:
```
string source = """app.MapGet("/", () => TypedResults.Extensions.TestResult();""";
```
But I don't know where I can add the `TestResult` extension method.

Reply:
You can add the definition here: https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTestBase.cs#L253

That's the mock source file the `MapGet` code gets written into.

Reply:
I was just checking if there was not other obvious place I missed :)